Agoda.Frameworks.DB/DbRepository_New.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Agoda.Frameworks.LoadBalancing;
+using Dapper;
+
+
+namespace Agoda.Frameworks.DB
+{
+    public partial class DbRepository // new simplified stuff
```

Comment:
What is your reasoning to split these into a partial class?

Reply:
the class is too big

Reply:
Not a fan of the reasoning, but OK :)